cmd/protoc-gen-go-grpc/protoc-gen-go-grpc_test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex # Debugging enabled.
+
+die() {
+    echo "$@" >&2
+    exit 1
+}
+
+trap "git clean --force --quiet" EXIT
+
+WD="$(dirname $0)"
+
+protoc \
+    --go_out=. \
+    --go_opt=paths=source_relative \
```

Comment:
I don't believe this is strictly necessary since we never actually compile the pb.go files.

Reply:
Done